frontend/src/app/sessions/session/session.component.css

```diff
@@ -7,6 +7,10 @@
   height: calc(100vh - 2vh - 65px - 110px);
 }
 
+.heigh-fullscreen {
+  height: calc(100vh - 2.6vh);
+}
+
 .iframe-container {
   position: relative;
 }
```

Comment:
As far as I can see, this has no effect. We should be able to remove it from CSS & HTML.

Reply:
It had an effect on my side but I changed it to a tailwind class so removed it here.